### DISCUSSION TITLE
Now that the FAQ/Forum has moved to GH Discussions, shouldn't it be changed on faq.i3wm.org?

### DISCUSSION THREAD

**Discussion by lxhom** [i3/i3#4512] [Ideas]
Hi, i just stumbled upon a faq.i3wm.org page while configuring i3, and it says that it moved to reddit. But now that the official channel is GH Discussions, shouldn't it have a banner that links to here?
![2021-09-12-014539_852x222_scrot](https://user-images.githubusercontent.com/73352734/132966196-f6737397-e7b3-48fb-8b69-a8541a05d296.png)

**Comment by Airblader** [comment]
Thanks! See https://github.com/i3/i3/issues/4400#issuecomment-917868199

**Comment by stapelberg** [comment]
Fixed with https://github.com/i3/faq.i3wm.org/commit/35426aad4debabe2f107465fb3128977cd941e09
